modules/distr-tracing-tempo-getting-the-tempostack-instance-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="getting-tempo-stack-logs_{context}"]
+= Getting the TempoStack instance logs
+
+You can get the logs from a TempoStack component as follows.
+
+.Procedure
+
+. Locate the component you want to get the logs for. You can do this by listing all the deployments on an specific namespace.
```

Comment:
It's not clear if a user would know how to do this - maybe if there is an `oc` CLI command they can use to do this we should include it here, and an example output?

Reply:
@rubenvp8510, WDYT?

Reply:
Yes, 
I think the standard way of doing this is using:
  `oc get pods` 
to list all pods

Then select the ones that belongs to the component you want to inspects logs for

oc logs <pod name> , usually the pod name follows this pattern : `tempo-<tempo-stack-name>-<component>`